builtin/providers/aws/resource_aws_elb_load_balancer_policy.go

```diff
@@ -0,0 +1,350 @@
+					Schema: map[string]*schema.Schema{
+						"name": &schema.Schema{
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+
+						"value": &schema.Schema{
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+					},
+				},
+			},
+		},
+	}
+}
+
+func resourceAwsLoadBalancerPolicyCreate(d *schema.ResourceData, meta interface{}) error {
+	elbconn := meta.(*AWSClient).elbconn
+
+	attributes := []*elb.PolicyAttribute{}
+	if attributedata, ok := d.GetOk("policy_attribute"); ok {
+		attributeSet := attributedata.(*schema.Set).List()
+		for _, attribute := range attributeSet {
+			data := attribute.(map[string]interface{})
+			attributes = append(attributes, &elb.PolicyAttribute{
+				AttributeName:  aws.String(data["name"].(string)),
+				AttributeValue: aws.String(data["value"].(string)),
+			})
+		}
+	}
+
+	lbspOpts := &elb.CreateLoadBalancerPolicyInput{
+		LoadBalancerName: aws.String(d.Get("load_balancer_name").(string)),
+		PolicyName:       aws.String(d.Get("policy_name").(string)),
+		PolicyTypeName:   aws.String(d.Get("policy_type_name").(string)),
+		PolicyAttributes: attributes,
+	}
+
+	if _, err := elbconn.CreateLoadBalancerPolicy(lbspOpts); err != nil {
+		return fmt.Errorf("Error creating LoadBalancerPolicy: %s", err)
+	}
+
+	d.SetId(fmt.Sprintf("%s:%s",
+		*lbspOpts.LoadBalancerName,
+		*lbspOpts.PolicyName))
+	return nil
+}
+
+func resourceAwsLoadBalancerPolicyRead(d *schema.ResourceData, meta interface{}) error {
+	elbconn := meta.(*AWSClient).elbconn
+
+	loadBalancerName, policyName := resourceAwsLoadBalancerPolicyParseId(d.Id())
+
+	request := &elb.DescribeLoadBalancerPoliciesInput{
+		LoadBalancerName: aws.String(loadBalancerName),
+		PolicyNames:      []*string{aws.String(policyName)},
+	}
+
+	getResp, err := elbconn.DescribeLoadBalancerPolicies(request)
+	if err != nil {
+		if ec2err, ok := err.(awserr.Error); ok && ec2err.Code() == "PolicyNotFound" {
+			d.SetId("")
+			return nil
+		}
+		return fmt.Errorf("Error retrieving policy: %s", err)
+	}
+
+	if len(getResp.PolicyDescriptions) != 1 {
+		return fmt.Errorf("Unable to find policy %#v", getResp.PolicyDescriptions)
+	}
+
+	policyDesc := getResp.PolicyDescriptions[0]
+	policyTypeName := policyDesc.PolicyTypeName
+	policyAttributes := policyDesc.PolicyAttributeDescriptions
+
+	attributes := []map[string]string{}
+	for _, a := range policyAttributes {
+		pair := make(map[string]string)
+		pair["name"] = *a.AttributeName
+		pair["value"] = *a.AttributeValue
+		if (*policyTypeName == "SSLNegotiationPolicyType") && (*a.AttributeValue == "false") {
+			continue
+		}
+		attributes = append(attributes, pair)
+	}
+
+	d.Set("policy_name", policyName)
+	d.Set("policy_type_name", policyTypeName)
+	d.Set("load_balancer_name", loadBalancerName)
+	d.Set("policy_attribute", attributes)
+
+	return nil
+}
+
+func resourceAwsLoadBalancerPolicyUpdate(d *schema.ResourceData, meta interface{}) error {
+	elbconn := meta.(*AWSClient).elbconn
```

Comment:
In the Update, we should make use of d.HasChange(""). That way we are only making an update when required